Sources/MapboxMaps/Gestures/GestureHandlers/PinchGestureHandlerImpl1.swift

```diff
@@ -40,7 +41,7 @@ internal final class PinchGestureHandlerImpl1: PinchGestureHandlerImpl {
         guard let view = gestureRecognizer.view else {
             return
         }
-        let pinchMidpoint = gestureRecognizer.location(in: view)
+        let pinchMidpoint = focalPoint ?? gestureRecognizer.location(in: view)
```

Comment:
`pinchMidpoint` is used for other purposes as well. This change should only apply when it's used for anchor.

Reply:
True, `pinchMidpoint` is used for zoom/rotation and dragging the map here. I think it is expected that the ability to pan with pinch gesture would be disabled when specifying a focal point(so to say drag happens from/to the specified focal point).
If we proceed with applying the focal point option only to zoom in pinch gesture. Then, in order to meet the feature request, we would need to add a separate option to disable panning while pinching. 

@pengdev how focal point handled on Android in this regard?

Reply:
I think the focal point doesn't affect pan gesture. 
And since we use one focal point for all gestures, we could always use the `pinchMidPoint` as the focal point?

Reply:
@ZiZasaurus is already working on adding a flag to disable panning during the pinch gesture: https://github.com/mapbox/mapbox-maps-ios/pull/1092

I agree that it wouldn't make much sense to use a non-nil focalPoint when panning is enabled. Should we add some logging when a gesture begins if we detect that condition?

Reply:
and focalPoint should only apply to the final of the 3 setCameras that happen in the `.changed` case of this handler.

Reply:
Made it so the focal point option affects only zoom and rotation. Added logging as well.